rollbar_dart/lib/src/sender.dart

```diff
@@ -1,6 +1,5 @@
-import 'api/response.dart';
-
 /// Sender interface to send payload data to Rollbar.
 abstract class Sender {
-  Future<Response?> send(Map<String, dynamic>? payload);
+  Future<bool> send(Map<String, dynamic>? payload);
```

Comment:
These method's return was a bit underdocumented before, when it returned the response, but it could be inferred. Now that it returns a bool, we really should document what true and false represent.